error/error.go

```diff
@@ -0,0 +1,13 @@
+package error
+
+import (
+    "fmt"
+    "os"
+)
+
+func DebugError(err interface{}) {
```

Comment:
You can use `panic()` for this. For dev process, you can use better tools like https://github.com/davecgh/go-spew which will give more detailed information about the error